lib/active_fedora/aggregation/association.rb

```diff
@@ -5,14 +5,8 @@ def ordered_reader
       OrderedReader.new(owner).to_a
     end
 
-    def add_link(proxy)
-      LinkInserter.new(owner, proxy).call
-    end
-
-    def save_through_record(record)
```

Comment:
This is called by ActiveFedora::Associations::IndirectlyContainsAssociation#insert_record(). Are you sure you have test coverage for insert_record()?